bin/etcd-manage

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -eo pipefail
+
+preStart() {
+  # pull ip address for current machine and set in environment
+  export ETCD_IP=$(hostname -i)
+  _log "Started etcd at ${ETCD_IP}"
+}
+
+# TODO: Need to stop etcd properly
```

Comment:
`member remove` is what I'd like to utilize but I don't have a shutdown hook at the moment having removed ContainerPilot.